Systems/carburetor.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0"?>
+
+<!--
+***********************************************************************************
+Carburator icing and carb heat toggle
+***********************************************************************************
+ -->
+
+<system name="carburetor">
+
+    <channel name="icing">
+    
+        <fcs_function name="Carb heat on cuases RPM drop on engine 0">
+            <function>
+                <ifthen>
+                    <property>/controls/anti-ice/engine[0]/carb-heat</property>
+                    <value>0.35</value>
+                    <value>0.15185</value>
+                </ifthen>
+            </function>
+            <output>/fdm/jsbsim/propulsion/engine[0]/air-intake-impedance-factor</output>
+            <output>/fdm/jsbsim/propulsion/engine[1]/air-intake-impedance-factor</output>
```

Comment:
I am using the new alias inside the `<ifthen>` block and I also now have two __proper__ functions, sorry for the mess. But what happened before was that the `carb-heat[0]` is being used by both engines, and I forgot to clean this messy part of the code (I was troubleshooting why the carb heat wasn't working for the 180 HP and I left this mess here). I now made this code look pretty, but I don't know where could I fix the problem (that is, where can I associate the 180 HP to `carb-heat[1]`)